src/map/ai/states/magic_state.h

```diff
@@ -60,7 +60,7 @@ class CMagicState : public CState
     void   ApplyEnmity(CBattleEntity* PTarget, int ce, int ve);
     void   ApplyMagicCoverEnmity(CBattleEntity* PCoverAbilityTarget, CBattleEntity* PCoverAbilityUser, CMobEntity* PMob);
 
-    void SetInstantCast(const bool bInstantCast)
+    void SetInstantCast(bool const bInstantCast)
```

Comment:
Let it be now and forever known. I hate this. I don't want to live on this planet.